app/src/services/fsManagerServiceAdapter.ts

```diff
@@ -5,7 +5,6 @@ import {
   Collection,
   EnvironmentEntity,
   ErroredRecord,
-  FileSystemResult,
```

Comment:
Why remove it? If we already have `FileSystemResult` and `FileSystemError` then we can use it